extensions/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>io.smallrye</groupId>
+        <artifactId>smallrye-config-parent-1.3</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    
+    <groupId>io.smallrye.ext</groupId>
+    <artifactId>smallrye-config-extensions-1.3</artifactId>
+    
+    <packaging>pom</packaging>
+    <name>SmallRye: MicroProfile Config Extensions</name>
+    
+    <modules>
+        <module>converters/json</module>
+    </modules>
+    
+    <properties>
+        <microProfile.version>2.2</microProfile.version>
```

Comment:
This version should be removed, don't think it's used

Reply:
Yes, not yet. Sorry it will come in the subsistent PR's. Will remove for now.